fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/app/api/os/VehicleOperatingSystem.java

```diff
@@ -171,8 +170,7 @@
      * Activates the detection of the leading vehicle within in a given distance.
      *
      * @param sensorRange The maximum distance to look ahead for leading vehicles.
-     * @param sensorType  The type of the sensor (e.g. LIDAR)
+     * @param sensors     List of distance sensors
```

Comment:
The previous version of this comment was correct, please check (also `sensorRange` description should be more general)